src-backbone/app/js/views/builder/pageDetails/pageElements/elementsCompositeView.js

```diff
@@ -16,4 +23,14 @@ module.exports = Marionette.CompositeView.extend({
         this.collection = this.model.elements;
     },
 
+    _onCreateNewElement: function(event) {
+        event.preventDefault();
+
+        var modalView = new ModalLayoutView({
+            title: i18n.t('New Element'),
+            bodyView: new ConceptSearchModalView({ page: this.model }),
+        });
+        App().RootView.modal.show(modalView);
```

Comment:
why not create a helper function for RootLayoutView called `showModal` like `showNotification`?